packages/bpk-component-chip-group/src/BpkChipGroupSingleSelect.tsx

```diff
@@ -18,14 +18,13 @@
 
 import { useState } from 'react';
 
-import BpkChipGroup, { type ChipItem, type SingleSelectChipItem, type CommonProps } from './BpkChipGroup';
+import BpkChipGroup, { type ChipItem, type SingleSelectChipItem, type ChipGroupProps } from './BpkChipGroup';
 
 export type SingleSelectProps = {
   chips: SingleSelectChipItem[];
   selectedIndex?: number;
   onItemClick?: (item: SingleSelectChipItem, selected: boolean, index: number) => void,
-} & CommonProps;
-
+} & ChipGroupProps;
```

Comment:
Want to use the entire ChipGroupProps because consumer can choose `type` and then may need to provided specific args based on selection

Reply:
I think this might have cause the `chips` prop to now include the `ChipItem` as well as `SingleSelectChipItem`
<img width="723" alt="image" src="https://github.com/Skyscanner/backpack/assets/22656572/22a89835-98bc-4d79-82f6-ca3248493fbf">